external/CMakeLists.txt

```diff
@@ -325,6 +325,10 @@ target_include_directories(libatrac9 PUBLIC LibAtrac9/C/src)
 
 option(BUILD_SHARED_LIBS "Build shared library" OFF)
 option(XXHASH_BUILD_XXHSUM "Build the xxhsum binary" OFF)
+option(XXH_X86DISPATCH_ALLOW_AVX "Allow building XXH3 with AVX even if it crashes on SSE2-Only CPUs" OFF)
+if(XXH_X86DISPATCH_ALLOW_AVX)
+	target_compile_definitions(xxhash PRIVATE XXH_X86DISPATCH_ALLOW_AVX)
+endif()
```

Comment:
xxhash is not defined at this point. I'm not entirely sure but I think doing so here would cause CMake to fail.
This needs to be added after the call to add_subdirectory